### PR TITLE
Bugsnag

### LINKED ISSUE #909: Name History Icon not shown

### LINKED ISSUE #938: Integrate Bugsnag
https://www.bugsnag.com/ free for open source projects

### PR DESCRIPTION
**Pull request by 1-alex98**
Fixes #938

### CODE REVIEW

**Comment by 1-alex98** [comment]
@micheljung you undrstand what travis problem is?

**Comment by norraxx** [comment]
Can we somehow "push" @axel1200 work forward? It would be nice to have such information, especially now.

**Comment by 1-alex98** [comment]
No you can not because i was told we dont want this.... so I stoped working on it. Go punsh @Brutus5000  and @micheljung and tell them you want it

File: src/main/resources/i18n/messages.properties [hunk 1]
```diff
@@ -653,3 +653,12 @@ tournament.openOnChallonge=Open on challonge.com
 tournament.gameType=Game type:
 tournament.startingAt=Starting at:
 tournament.completedAt=Completed at:
+
+report.logLinesToBeSent=Log lines to be sent
+report.feedback=Tell us what happend or what is wrong:
+report.contect=Feel free to contact me
```
Comment (1-alex98) [inline src/main/resources/i18n/messages.properties]: delete

**Comment by 1-alex98** [comment]
Atm i send errors silently and also I send the username of the user. In terms of privacy this seems problematic how should i do it @Geosearchef ???

**Comment by 1-alex98** [comment]
Tell me how to do it

**Pull request closed without merging**